tests/components/integration/test_init.py

```diff
@@ -121,3 +121,89 @@ def _get_device_config_entries(entry: er.RegistryEntry) -> set[str]:
     # Check that the config entry association has updated
     assert config_entry.entry_id not in _get_device_config_entries(input_entry)
     assert config_entry.entry_id in _get_device_config_entries(valid_entry)
+
+
+async def test_device_cleaning(hass: HomeAssistant) -> None:
+    """Test for source entity device for Trend."""
+    device_registry = dr.async_get(hass)
+    entity_registry = er.async_get(hass)
```

Comment:
Just noticed this one, which probably also applies to the once we just merged 🙈 

```suggestion
async def test_device_cleaning(hass: HomeAssistant, device_registry: dr.DeviceRegistry, entity_registry: er.EntityRegistry) -> None:
    """Test for source entity device for Trend."""
```

Reply:
Changed here and I will also do it in the others.